### ISSUE TITLE
People of WordPress - September 2022

### ISSUE THREAD

**Issue by abhansnuk** [WordPress/Marketing-Team#104]

**Comment by nalininsbs** [comment]
This feature will be moved as the original is not live on HeroPress. 

The story for September 2022 to coincide with International Translation Day will be Huanyi (Eric) Chuang. Change confirmed with @Bringmannn and spreadsheet updated.
[Original HeroPress article](https://heropress.com/essays/connection-makes-credibility-credibility-makes-opportunity/) 

**Summary from the document**
Original review, questions, and structure: sthakor - done
Project leading and setting up arrangements: @abhansnuk, @meherbala (for Larissa) - done
Liaison with interviewee, taking questions list and obtaining photos: Adeeb Malik and @murillol - in progress. Adeeb to speak with Huanyi, Wednesday 17 August 2022. A colleague of Adeeb's may also be contributing alongside him.
Deadline for first draft of feature: 22 August 2022

**Comment by Ad-Mk** [comment]
thanks @nalininsbs

**Comment by murillol** [comment]
Hi @nalinisbs! I'm not sure what this means: 

> Project leading and setting up arrangements: @abhansnuk, @meherbala (for Larissa)

Was I supposed to do something here and I missed it? If so, sorry! 

I've checked the planning sheet and it doesn't look like I'm assigned anything for the last quarter, but just in case there are changes, I do **not have availability until 17 October**

**Comment by meherbala** [comment]
Hi @Ad-Mk please can you ask Huanyi (Eric) Chuang to provide 3-4 pictures for the article. 

Featured image for any article for POWP is 1247px by 627px. 

Please can you make sure that out of the 3-4 pictures at least 1 image size needs to be 1,500px (width)? 

This will help me make that image as the main featured image.

Thank You.

**Comment by Ad-Mk** [comment]
Hi @meherbala, I'll do that, thanks!

**Comment by abhansnuk** [comment]
Hello @Ad-Mk Would you be able to share your update from the online chat with Huanyi on the questions we shared, and how you are getting on with the draft for the first deadline please. Have you been able to upload pictures to the folder please? Thanks.

**Comment by Ad-Mk** [comment]
Hi @Abha,

I have shared the questions and picture requirements with Eric. I’m still
waiting on his response. As soon as I get those, the draft won’t take more
than a day.

**Comment by abhansnuk** [comment]
@murillol thanks for the question above , I think Nalini had added (for Larissa) to reflect that this is normally an area you are involved with and you had said you would not be able to work on liaising with the person featured for the September's edition already.

> Hi @nalinisbs! I'm not sure what this means:
> 
> > Project leading and setting up arrangements: @abhansnuk, @meherbala (for Larissa)
> 
> Was I supposed to do something here and I missed it? If so, sorry!
> 
> I've checked the planning sheet and it doesn't look like I'm assigned anything for the last quarter, but just in case there are changes, I do **not have availability until 17 October**

@murillol thanks for the question above , I think Nalini had added (for Larissa) to acknowledge that this is normally an area you are involved with, and you had said you would not be able to work on liaising with the person featured for the September's edition already. Meher said she would step in for some of the arrangements, and Adeeb has taken our questions to Huanyi.

Can I check on the date you mentioned, will you not be available to work on any other PofWP items from now until 17 October? We can adjust the next few editions accordingly. Thanks Larissa.

**Comment by abhansnuk** [comment]
Thanks @Ad-Mk . We had hoped for a first draft by 22 August. Would you be able to follow up with Huanyi please and see if there is anything we can help him with or if he would prefer to go through the additional questions in another format? Thanks.

**Comment by Ad-Mk** [comment]
I’ve already sent a message. Waiting to hear back. Till then if there’s
another PofWP feature. Do let me know.

**Comment by murillol** [comment]
> @murillol thanks for the question above , I think Nalini had added (for Larissa) to acknowledge that this is normally an area you are involved with, and you had said you would not be able to work on liaising with the person featured for the September's edition already. Meher said she would step in for some of the arrangements, and Adeeb has taken our questions to Huanyi.
> 
> Can I check on the date you mentioned, will you not be available to work on any other PofWP items from now until 17 October? We can adjust the next few editions accordingly. Thanks Larissa.

Thanks for clarifying @abhansnuk! Yes that's exactly right, I will be on holiday and then travelling for work in September and October, and am back on a normal schedule starting on 17 October. During this time I will be involved in the weekly marketing meeting as much as I can, but will have a very unpredictable schedule - which is why I think I will not be able to make PoW deadlines.

**Comment by meherbala** [comment]
Hi @Ad-Mk as we are already in Sep any update on the POWP article would be appreciated.

**Comment by Ad-Mk** [comment]
Hi @meherbala, I've put up the article on the doc link. I'll be doing final edits and check. Will update asap!

**Comment by meherbala** [comment]
Thanks, @Ad-Mk.

**Comment by nalininsbs** [comment]
@Ad-Mk I am looking to go through the photos you had obtained and were going to add the link to this GitHub card and tag in @meherbala . Would you be able to give me the link to that message please? Thank you for following up the images.

**Comment by meherbala** [comment]
@nalininsbs I have already given @Ad-Mk the Google drive folder link to upload the photos. He has already uploaded 3 images.

Here is the link to the folder: https://drive.google.com/drive/u/1/folders/1dVC7MbFH-00jGCE__aDsGt9r,_r12UoS3

Currently Adeeb has uploaded are 3 images. I will work on them and upload the edited ones directly to the CMS by Wednesday OR Thursday.

Once he has the content finalized I will understand if we need more images for the post or not.

**Comment by meherbala** [comment]
I have edited the 4 images and uploaded to the News Dashboard.

Thank you to @Ad-Mk for co-ordinating with Huanyi (Eric) Chuang for the pictures.

CC: @nalininsbs

**Comment by nalininsbs** [comment]
@abhansnuk has interviewed Huanyi and is to work on the feature once the other marked updates are complete. Abha can you please liaise with @meherbala today to confirm if any more pictures might be needed. Thank you and to @meherbala for working on editing the pictures so far with Abha, and to Adeeb for following up the photos.

**Comment by meherbala** [comment]
@nalininsbs I have received 4 images and that is all Huanyi can provide. 

I think it should work.

All the images have been edited and uploaded to the CMS. Waiting for the content to be finalized and once on the CMS, I can add the images wherever needed.

**Comment by marybaum** [comment]
Apparently copy editing is in progress.

**Comment by nalininsbs** [comment]
This has been in the P2 from end of September awaiting one check. Once resolved, the post should be rechecked with links and photos checked before publication. Message also with Chloe on a particular question.

**Comment by nalininsbs** [comment]
Thank you for updates on the November scheduled item - photos and other information is awaited. Thank you @meherbala, @marybaum , @abhansnuk , @murillol for updates on the card. 

As previously discussed with @Bringmannn we will use this postponed September feature in November. Thank you @meherbala for time on this the last few days.

**Comment by abhansnuk** [comment]
Final checks taking place - WordPress IDs added. Meher reviewing any notes still in the post and images in preview. Mary Baum to recheck too. 

Tags to be rechecked. All Chloe et al's previous comments checked are incorporated.
https://wordpress.org/news/?p=13562&preview=1&_ppp=ce7b145bfa

**Comment by abhansnuk** [comment]
Link to published feature: https://wordpress.org/news/2022/11/people-of-wordpress-huanyi-chuang/

**WordPress.org IDs involved in work on this.**
Huanyi: no249a002
Writers, interviewers, fact-checkers, editors: webcommsat, marybaum, meher, cbringmann, sthakor, adeebmalik
Series thanks: chanthaboune, topher1kenobe

**Comment by abhansnuk** [comment]
To do: 
- Huanyi will send over a picture in Taiwan to swap for the picture he had from Australia
- translation: follow-up with Huanyi

**Comment by abhansnuk** [comment]
To send link to WCAsia: requested links to contributors from Asia for them to use for promotions.

**Comment by abhansnuk** [comment]
To keep open as new pictures coming in from Taiwan to be swapped in.

Follow-ups in WCAsia.

Local language translation query. This may have to be after WCAsia.

**Comment by abhansnuk** [comment]
Moving this to the done column. Please do not close until we have the link to the translation, and an updated image size on the Taiwan picture to be added to this.

**Comment by nalininsbs** [comment]
Reviewing. The photo to be changed to be done as not showing. @meherbala or @marybaum can you follow instead please as looks like not been changed to the photo sent from Taiwan or it did not fit.
@abhansnuk can you check on translation. 
@meherbala With WordCamp Asia in Taiwan, this may be useful to organisers.
Thank you.

**Comment by abhansnuk** [comment]
Is this still looking for a translator? @Ad-Mk would you like to follow this up or see if we can list it as available to translate with the relevant locale please? Are you going to WCAsia? Thanks

**Comment by meherbala** [comment]
in which month are we scheduling this?

**Comment by abhansnuk** [comment]
Hi @meherbala there is no set time for publishing the translation. We had been looking at which pending translations could be usefully highlighted at WCAsia Contributor Day perhaps on polyglots and marketing tables. We can share a list then with those table needs where there is not already actively working on a translation.
